.github/workflows/test.yml

```diff
@@ -0,0 +1,58 @@
+name: Run Tests
+on:
+  push:
+    branches: [ '*' ]
+  pull_request:
+    branches: [ main ]
+  schedule:
+    # Run automatically at 8AM PST Monday-Friday
+    - cron: '0 15 * * 1-5'
+  workflow_dispatch:
+
+jobs:
+  tests:
+    name: Build & Test
+    runs-on: ubuntu-latest
+    timeout-minutes: 20
+    strategy:
+      matrix:
+        go: [ '1.11', '1.12', '1.13', '1.14', '1.15', '1.16' ]
+    steps:
+      - name: Checkout smtpapi-go
+        uses: actions/checkout@v2
+
+      - name: Setup Go environment
+        uses: actions/setup-go@v2
+        with:
+          go-version: ${{ matrix.go }}
+
+      - name: Set GOPATH
+        run: echo "GOPATH=$HOME" >> $GITHUB_ENV
+      - name: Set GOBIN
+        run: echo "GOBIN=$GOPATH/bin" >> $GITHUB_ENV
+      - name: Set GO111MODULE
+        run: echo "GO111MODULE=off" >> $GITHUB_ENV
+
+      - name: Build smtpapi-go
+        run: make install
```

Comment:
`make test` already runs `make install` so this will do the install twice. Another option is to do both explicitly in 1 command which will just run install once: `make install test`

Reply:
Seems like this applies to both rest and smtp (but not sg-go), so I can just wipe the `make install` step from those two

Reply:
Update: this didn't work. Time for `make install test` to come to the rescue